farm/infer.py

```diff
@@ -54,9 +56,28 @@ def __init__(self, load_dir, batch_size=4, gpu=False):
             self.label_map = self.processor.label_maps[0]
         elif len(self.model.prediction_heads) == 0:
             self.prediction_type = "embedder"
-        self.name = os.path.basename(load_dir)
+        # self.name = os.path.basename(load_dir)
         set_all_seeds(42, n_gpu)
 
+    @classmethod
+    def load_from_dir(cls, load_dir, gpu=False):
```

Comment:
Please name it "load" to be more consistent with the rest of the package

Reply:
`load_from_dir` can be found in `processor.py` and it is also mentioned in `prediction_head.py` (but seems like an outdated comment).